sematic/resolvers/local_resolver.py

```diff
@@ -193,13 +197,25 @@ def _future_did_resolve(self, future: AbstractFuture) -> None:
     def _future_did_fail(self, failed_future: AbstractFuture) -> None:
         super()._future_did_fail(failed_future)
 
-        run = self._get_run(failed_future.id)
+        run = api_client.get_run(failed_future.id)
```

Comment:
Can you explain this change?

Reply:
I think it can go back to `self._get_run` now that cloud_resolver is overriding it